### ISSUE TITLE
Documentation Examples Not Working

### ISSUE THREAD

**Issue by LingamYoni** [AngusJohnson/Image32#78]
Hello,
I've been following the documentation examples provided at the following links:
http://www.angusj.com/image32/Docs/QuickStart.htm
http://www.angusj.com/image32/Docs/Units/Img32.Extra/Routines/Draw3D.htm
Unfortunately, none of these examples work as expected because the methods and functions mentioned seem to be unknown or missing. I am particularly interested in drawing the 3D text as shown in the image at http://www.angusj.com/image32/Images/text_3D.png. Could you please provide guidance on how to achieve this?

Thank you for your assistance.

**Comment by AngusJohnson** [comment]
Thank you for the feedback.
I've just checked **all** the sample code in QuickStart and I have found 2 errors that I'll fix shortly.

In the meantime, here are the 2 fixes ...
In the first **polygon rendering** example change the following line
```
    //rec := Img32.Vector.InflateRect(rec, -50,-50);
    Img32.Vector.InflateRect(rec, -50,-50);
```
In the second **polygon rendering** example change the following line
```
    //impact48.GetTextGlyphs(20, 130, 'Image32'); 
    impact48.GetTextOutline(20, 130, 'Image32'); 
```

**Comment by LingamYoni** [comment]
In the documentation, the method `FontManager.Load('Impact', 800)` is mentioned, but the method declaration for `TFontManager.Load` only includes a single parameter:

```pascal
function TFontManager.Load(const fontName: string): TFontReader;
```

Could you please clarify what the `800` parameter represents in the context of `FontManager.Load('Impact', 800)`? The method signature does not seem to support this additional parameter.

Thank you!

**Comment by AngusJohnson** [comment]
> Could you please clarify what the 800 parameter represents

Font Weight 😁.

> since the GetFont method does not exist in FontManager.

It does exist in FontManager, it just hasn't until now made it into the documentation 😜.

https://www.angusj.com/image32/Docs/Units/Img32.Text/Classes/TFontManager/Methods/Load.htm

**Comment by LingamYoni** [comment]
In the **latest** available release of Image32 version 4.3 on GitHub, the `GetFont` method is missing from `FontManager`. Additionally, the `FontManager.Load` method is only available with a single parameter.


Also, it might be time for an updated release. Your website's [Overview page](https://www.angusj.com/image32/Docs/Overview.htm) proudly proclaims version 4.4, but it appears to be a phantom release. 👻

**Latest Version:** 4.4  
**Last Update:** 2 May 2024

Could we get the real version 4.4 to match the site? 🙏

Thanks a bunch!

**Comment by AngusJohnson** [comment]
> In the **latest** available release of Image32 version 4.3 on GitHub, the `GetFont` method is missing from `FontManager`. 

https://github.com/AngusJohnson/Image32/blob/2285272ecca9bfbd50481f0cf5a45f2f1e351b4b/source/Img32.Text.pas#L281

Additionally, the `FontManager.Load` method is only available with a single parameter.

https://github.com/AngusJohnson/Image32/blob/2285272ecca9bfbd50481f0cf5a45f2f1e351b4b/source/Img32.Text.pas#L282-L284

**Comment by LingamYoni** [comment]
Dear @AngusJohnson ,

Please read carefully – this is about the **latest available release of Image32 version 4.3 on GitHub**.

In this release, the GetFont method is missing from FontManager. Additionally, the FontManager.Load method is only available with a single parameter:
![s](https://github.com/AngusJohnson/Image32/assets/40486018/879cfc7c-390c-4ba3-bd7a-ba3233437a4d)

**Comment by AngusJohnson** [comment]
> Please read carefully – this is about the latest available release of Image32 version 4.3 on GitHub.

I'm well aware that the "released" version is well out of date.
However, surely you (and most developers who download from GitHub) understand that if things aren't working as expected, then first thing to do is to download the latest repository revision. 
I write this stuff as a hobby, and if I had plenty of time, yes I would love to update the latest revision more frequently.
However, as it happens I'm very occupied with other things, and so I have to prioritise what I believe is most important with the time I do allocate to supporting this library. And ISTM that most people would prefer bug fixes in the repository over frequently updated "releases".

**Comment by LingamYoni** [comment]
Dear Angus,

Thank you so much for your detailed and thoughtful response. I completely understand and appreciate the immense effort you put into this project, especially as a hobby. I always make sure to download the latest repository revision and truly value the bug fixes and updates you provide there.

I apologize if my previous message came across as anything more than pointing out a minor inconsistency. Your work on Image32 is incredibly valuable, and I'm genuinely grateful for all the time and dedication you invest in it.

Thank you once again for your hard work and support!

Best regards
